modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -49,4 +49,21 @@
       end
     end
   end
+
+  factory :auto_established_claim_without_flashes_or_special_issues, class: 'ClaimsApi::AutoEstablishedClaim' do
+    id { SecureRandom.uuid }
+    status { 'pending' }
+    source { 'oddball' }
+    evss_id { nil }
+    auth_headers { { test: ('a'..'z').to_a.shuffle.join } }
+    form_data do
+      json = JSON.parse(File.read("#{::Rails.root}/modules/claims_api/spec/fixtures/form_526_no_flashes_json_api.json"))
```

Comment:
*nit
Maybe consider changing the JSON filename to `form_526_no_flashes_no_special_issues_json_api.json`?
Just because the only place it's referenced is in this factory named `auto_established_claim_without_flashes_or_special_issues`.